state_machine/drivers/emulation/lib/pycubed.py

```diff
@@ -43,26 +45,34 @@ def __init__(self):
         self.c_state_err = 0
         self.c_boot = None
 
+        schedule = tasko.schedule
+        rt = reader_task(self)
+        self.reader_task = schedule(5, rt.main_task, 10)  # try to read from stdin at 5Hz
+
+        self._accel = (0.0, 0.0, 0.0)
+        self._mag = (0.0, 0.0, 0.0)
+        self._gyro = (0.0, 0.0, 0.0)
+
     def new_file(self, substring, binary=False):
         print(
-            f"new file not implemented, {substring} not written with binary={binary}")
+            f"new file not implemented, string not written with binary={binary}")
         return None
 
     @property
     def acceleration(self):
-        return (0.0, 0.0, 0.0)
+        return self._accel
 
     @property
     def magnetic(self):
-        return (0.8, 0.3, -0.3)
+        return self._mag
 
     @property
     def gyro(self):
-        return (0.3, 0.0, 0.6)
+        return self._gyro
 
     def log(self, str):
         """Logs to sd card"""
-        print(f'log not implemented, tried to log {str}')
+        print('log not implemented')
 
 
 pocketqube = Sattelite()
```

Comment:
Sattelite -> Satellite

May want to fix this typo (just for the sake of consistency in all the versions of pycubed.py).

Reply:
Thanks for catching that